subscriptions/conf.py

```diff
@@ -24,9 +24,10 @@ class Currency():
             n_sep_by_space (bool): Whether the currency symbol is
                 separated from negative values by a space.
             mon_decimal_point (str): The character used for decimal
-                points.
+                points. Use 0 if the currency doesn't need fraction.
```

Comment:
Minor text fix: can we change this to "Use 0 if this is a non-decimal currency."

Reply:
Yes it would be better.